reporter-web-app/build.gradle

```diff
@@ -8,3 +8,15 @@ node {
 
     nodeModulesDir = file("${project.projectDir}/node")
 }
+
+task install(type: YarnTask) {
+    description 'Run yarn install to install the node dependencies. Similar to the yarn task but with up-to-date ' +
```

Comment:
Shorter: 'Use Yarn to install the Node dependencies. Similar to the "yarn" task but with up-to-date checks."